RentACar/RentACar/Helpers/CrudHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+
+namespace RentACar.Utilities
+{
+    static public class CrudHelper
+    {
+        public static Car UpdateCarInListById(List<Car> cars, Car car)
+        {
+            var carUpdated = cars.FirstOrDefault(e => e.Id == car.Id);
+
+            carUpdated.Brand = car.Brand;
+            carUpdated.Color = car.Color;
+            carUpdated.DoorsQuantity = car.DoorsQuantity;
+            carUpdated.Model = car.Model;
+            carUpdated.Transmission = car.Transmission;
+
+            return carUpdated;
+        }
+        public static int GetNewId(List<Car> cars)
```

Comment:
remember to leave one empty line between methods so it's easy to know where a method ends (improves legibility)

Reply:
Oh, yes! Thanks for reminding me of that!